test/basic-atomic-error-test.js

```diff
@@ -17,16 +16,24 @@ test('test error in previous query', assert => {
   domain1.run(() => {
     return db.atomic(() => {
       const first = db.getConnection().then(conn => {
-        return Promise.promisify(conn.connection.query)('ONE')
-          .then(() => conn.release())
+        return new Promise((resolve, reject) => {
```

Comment:
I recommend to use `const { promisify } = require('utils')` to promisify, we should very rarely call `new Promise`.